components/LifeAtTorchbox/VideoPlayPauseButton.tsx

```diff
@@ -0,0 +1,52 @@
+import styles from '../../styles/VideoPlayPauseButton.module.scss';
+
+type VideoPlayPauseButtonProps = {
+    playing: boolean;
+    toggleVideoPlaying: () => void;
+    className?: string;
+};
+
+export const VideoPlayPauseButton = ({
+    className,
+    toggleVideoPlaying,
+    playing,
+}: VideoPlayPauseButtonProps) => {
+    const handleKeyboardEvent = (event: React.KeyboardEvent<SVGSVGElement>) => {
+        //Arrow keys can be used to scroll the webpage.
+        if (!['Tab', 'Shift', 'ArrowDown', 'ArrowUp'].includes(event.key)) {
+            toggleVideoPlaying();
+        }
+    };
+
+    return (
+        <svg
```

Comment:
This could be seperated into it's own icon component to make it reusable for future use.

Reply:
I've kept this svg as part of the play pause button as it already has several modifications made to it for the specific use of the video player.